scripts/production/gene_tree_stats.pl

```diff
@@ -137,24 +137,28 @@ sub get_gene_tree_cov {
     my $sorted_nodes = shift @_;
 
     my @gene_coverage = ();
-    my @sums = (0) x 6;
-    my @tags = qw(nb_genes nb_seq nb_orphan_genes nb_genes_in_tree nb_genes_in_tree_single_species
-                  nb_genes_in_tree_multi_species);
+    my @sums = (0) x 8;
+    my @tags = qw(nb_genes nb_seq nb_orphan_genes nb_genes_in_unfiltered_cluster nb_genes_unassigned
+                  nb_genes_in_tree nb_genes_in_tree_single_species nb_genes_in_tree_multi_species);
```

Comment:
This is another formatting opportunity to improve readability - each of these tags could have a line of their own!

Reply:
I've considered this suggestion, but I think I'll leave this as it is, on the basis that if there were a compelling case for it, it would already have been suggested during the review of [Compara PR 358](https://github.com/Ensembl/ensembl-compara/pull/358).